pymc_marketing/clv/plotting.py

```diff
@@ -344,15 +346,138 @@ def plot_probability_alive_matrix(
         ylabel=ylabel,
         title=title,
     )
-    force_aspect(ax)
+    _force_aspect(ax)
 
     # plot colorbar beside matrix
     plt.colorbar(pcm, ax=ax)
 
     return ax
 
 
-def force_aspect(ax: plt.Axes, aspect=1):
+def plot_expected_purchases(
+    model,
+    purchase_history: pd.DataFrame,
+    customer_id_col: str,
+    datetime_col: str,
+    t: int,
+    plot_cumulative: bool = True,
+    t_cal: int | None = None,
+    datetime_format: str | None = None,
+    time_unit: str = "D",
+    time_scaler: float | None = 1,
+    sort_transactions: bool | None = True,
+    set_index_date: bool | None = False,
+    title: str | None = None,
+    xlabel: str = "Time Periods",
+    ylabel: str = "Purchases",
+    ax: plt.Axes | None = None,
+    **kwargs,
+):
```

Comment:
Can we add the return type hint `plt.Axes` (or shall we return the Figure object instead?, we do this for the MMM plots)